.gitlab-ci.yml

```diff
@@ -171,7 +171,7 @@ build:macOS:
     - brew unlink python@2
     - brew install python3
     - pip3 install macpack
-    - opam init -a -y -j $NJOBS --compiler=${COMPILER} default https://opam.ocaml.org
+    - opam init -a -y -j ${NJOBS} --compiler=${COMPILER} default https://opam.ocaml.org
```

Comment:
This change doesn't do anything, try sticking a `printenv` to see what variables are defined.